### PR TITLE
Admin can create new tutorial

### PR DESCRIPTION
**Pull request by fentontaylor**
Adds the following:
- deletes dead tutorials index page
- fixes error handling with `current_admin?`, now renders 404 instead of raising an exception
- adds test for admin adding a new tutorial, including sad path
- adds validations (with test) for tutorials
- updates routes, Admin::TutorialController, and new tutorial form

### CODE REVIEW

File: spec/models/tutorial_spec.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe Tutorial, type: :model do
+  describe 'validations' do
+    it {should validate_presence_of :title}
+    it {should validate_presence_of :description}
+    it {should validate_presence_of :thumbnail}
```
Comment (nancylee713) [inline spec/models/tutorial_spec.rb]: To verify that a given thumbnail is "valid", does the attribute need stronger validation? For instance, a thumbnail should like something like this: `https://i.ytimg.com/vi/qMkRHW9zE1c/hqdefault.jpg`; checking for extension or file type?

**Comment by nancylee713** [comment]
Should we add a button that says "Add videos" somewhere on the page? So that the admin user can add videos after creating a brand new tutorial?

File: spec/models/tutorial_spec.rb (hunk 1 above)
Comment (nathangthomas) [reply spec/models/tutorial_spec.rb]: Good point. I don't think we need to be that stringent at this time. Let's keep it in mind for future refactors if time provides.

**Comment by nathangthomas** [comment]
> Should we add a button that says "Add videos" somewhere on the page? So that the admin user can add videos after creating a brand new tutorial?

We will need to in order to get one of our failing tests to pass. ```rspec ./spec/features/admin/edit_tutorial_spec.rb:7 # An Admin can edit a tutorial by adding a video```

**Pull request merged**